.changeset/blue-suits-drop.md

```diff
@@ -0,0 +1,5 @@
+---
+"@vue-storefront/middleware": minor
+---
+
+block overwriting alokai metadata
```

Comment:
All those changesets messages will appear in the final changelog, you should consolidate them into a single one or just remove those with messages like `rc`.